modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6924,7 +6924,7 @@ enable-in-x=Enable in {0}
 enable-index-actions-in-a-virtual-instance=Enable Index Actions in a Virtual Instance
 enable-index-actions-in-a-virtual-instance-help=Set the instance ID (i.e. company ID) of a virtual instance where administrators can reindex the instance in the Control Panel (Search > Index Actions).
 enable-inheritance=Enable Inheritance
-enable-inheritance-to-share-settings-between-related-data-models=Enable inheritance to share settings between related data models.
+enable-inheritance-to-share-settings-across-related-data-models=Enable inheritance to share settings across related data models.
```

Comment:
I think that the Figma messages are outdated and that the current (using "between") message is the right one. Have you confirmed this with the product people (@anabuchmann)? If not, please do it.